cli/cli_app.go

```diff
@@ -248,6 +251,21 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		}
 	}
 
+	if shouldPrintTerragruntInfo(terragruntOptions) {
```

Comment:
This can probably go earlier, before assume role?

Reply:
In fact, could this check go into `runCommand`?

Reply:
I don't claim to know where the best place to drop this logic into the call chain is.

If you think within `runCommand` is best, I'll take a look there today.

Reply:
@brikis98 I believe I put this `shouldPrintTerragruntInfo()` test at this point in `runTerragrunt` because
- we need to first call `getTerraformSourceUrl()`
- we then need to `downloadTerraformSource()`

both of which are needed in order to get the value of `WorkingDir` updated to include
the `.../.terragrunt_cache/...` in the path.

Reply:
OK, fair enough!